tests/test_image_seq2seq.py

```diff
@@ -61,10 +53,7 @@
 }
 
 
-@unittest.skip(
-    "need pytorch 1.4 release, https://github.com/pytorch/vision/issues/1712"
-)
-@unittest.skipUnless(TORCH_AVAILABLE, 'Must use torch 1.2 or above')
+@testing_utils.skipUnlessTorch14
```

Comment:
What does Torch 1.4 do that allows us to run these tests now?

Reply:
There was a bug in torch vision.